diff --git a/Vlv2TbltDevicePkg/PlatformPkgGccX64.dsc b/Vlv2TbltDevicePkg/PlatformPkgGccX64.dsc
index 1daf713..47e0722 100644
--- a/Vlv2TbltDevicePkg/PlatformPkgGccX64.dsc
+++ b/Vlv2TbltDevicePkg/PlatformPkgGccX64.dsc
@@ -1638,6 +1638,7 @@ MdeModulePkg/Universal/FaultTolerantWriteDxe/FaultTolerantWriteSmmDxe.inf
 
   Vlv2TbltDevicePkg/Application/FirmwareUpdate/FirmwareUpdate.inf
   Vlv2TbltDevicePkg/Application/SsdtUpdate/SsdtUpdate.inf
+  Vlv2TbltDevicePkg/Application/AudioSsdtUpdate/AudioSsdtUpdate.inf
 
   !if $(CAPSULE_ENABLE)
     MdeModulePkg/Universal/EsrtDxe/EsrtDxe.inf
diff --git a/Vlv2TbltDevicePkg/PlatformPkgX64.dsc b/Vlv2TbltDevicePkg/PlatformPkgX64.dsc
index e805871..efb8cbf 100644
--- a/Vlv2TbltDevicePkg/PlatformPkgX64.dsc
+++ b/Vlv2TbltDevicePkg/PlatformPkgX64.dsc
@@ -1651,6 +1651,7 @@ MdeModulePkg/Universal/FaultTolerantWriteDxe/FaultTolerantWriteSmmDxe.inf
 
   Vlv2TbltDevicePkg/Application/FirmwareUpdate/FirmwareUpdate.inf
   Vlv2TbltDevicePkg/Application/SsdtUpdate/SsdtUpdate.inf
+  Vlv2TbltDevicePkg/Application/AudioSsdtUpdate/AudioSsdtUpdate.inf
 
   !if $(CAPSULE_ENABLE)
     MdeModulePkg/Universal/EsrtDxe/EsrtDxe.inf